README.rst

```diff
@@ -125,37 +125,35 @@ Here is an example on how to use the ``GLM`` estimator.
 
     import numpy as np
     import scipy.sparse as sps
-    from sklearn.preprocessing import StandardScaler
-    from pyglmnet import GLM
+    from pyglmnet import GLM, simulate_glm
 
-    # create an instance of the GLM class
-    glm = GLM(distr='poisson')
-
-    n_samples, n_features = 10000, 100
+    n_samples, n_features = 1000, 100
+    distr = 'poisson'
 
-    # sample random coefficients
+    # sample a sparse model
     beta0 = np.random.normal(0.0, 1.0, 1)
     beta = sps.rand(n_features, 1, 0.1)
     beta = np.array(beta.todense())
 
-    # simulate training data
-    X_train = np.random.normal(0.0, 1.0, [n_samples, n_features])
-    y_train = glm.simulate(beta0, beta, X_train)
+    # simulate data
+    Xtrain = np.random.normal(0.0, 1.0, [n_samples, n_features])
+    ytrain = simulate_glm('poisson', beta0, beta, Xtrain)
+    Xtest = np.random.normal(0.0, 1.0, [n_samples, n_features])
+    ytest = simulate_glm('poisson', beta0, beta, Xtest)
 
-    # simulate testing data
-    X_test = np.random.normal(0.0, 1.0, [n_samples, n_features])
-    y_test = glm.simulate(beta0, beta, X_test)
+    # create an instance of the GLM class
+    glm = GLM(distr='poisson', score_metric='deviance')
 
     # fit the model on the training data
-    scaler = StandardScaler().fit(X_train)
-    glm.fit(scaler.transform(X_train), y_train)
+    glm.fit(Xtrain, ytrain)
```

Comment:
I also noticed that in the README, we still refer to the 'multinomial' option. We should remove it for now ...

Reply:
irrelevant after rebase